float64 x_lat
float64 y_long
float64 z_alt
